jobs/stackdriver-nozzle/templates/stackdriver-nozzle-ctl.erb

```diff
@@ -32,8 +32,8 @@ case $1 in
     echo $$ > $PIDFILE
 
     exec chpst -u vcap:vcap /var/vcap/packages/stackdriver-nozzle/bin/stackdriver-nozzle \
-      >>$LOG_DIR/stackdriver-nozzle.stdout.log \
-      2>>$LOG_DIR/stackdriver-nozzle.stderr.log
+      >> >(tee $LOG_DIR/stackdriver-nozzle.stdout.log) \
```

Comment:
the only oddity to consider here is that the messages from stackdriver-nozzle will get rolled up into the stackdriver-nozzle-ctl log file and will also show up as `vcap.stackdriver-nozzle-ctl` in syslog.

I don't think the syslog category matters but duplicating the logs is probably not necessary.